shortames.conf

```diff
@@ -1,4 +1,10 @@
 [aliases]
+  # centos
+  "centos" = "registry.centos.org/centos"
```

Comment:
Is there not a centos7 / centos8 image?

Reply:
You pull centos:7 or centos:8

They do not provide then as centos7 or centos8.

Reply:
Ack, OK